types/idle-js/index.d.ts

```diff
@@ -0,0 +1,89 @@
+// Type definitions for idle-js 1.2
+// Project: https://github.com/soixantecircuits/idle-js
+// Definitions by: Jeff Peirson <https://github.com/jpeirson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class IdleJs {
+    /**
+     * Creates a new unstarted idle tracking instance.
+     *
+     * @param options Initial options
+     * @throws
+     */
+    constructor(options?: IdleJs.Options);
+
+    /**
+     * Starts tracking idle state.
+     *
+     * @returns this instance for call chaining
+     */
+    start(): IdleJs;
+
+    /**
+     * Stops all tracking.
+     *
+     * @returns this instance for call chaining
+     */
+    stop(): IdleJs;
+
+    /**
+     * Reset visible and idle state.
+     *
+     * @param idle The new idle state, or undefined to use the existing state.
+     * @param visible The new visibility state, or undefined to use the existing state.
+     * @returns this instance for call chaining
+     */
+    reset(idle?: boolean, visible?: boolean): IdleJs;
```

Comment:
change `IdleJs` to `this`

```suggestion
    reset(idle?: boolean, visible?: boolean): this;
```